keadm/cmd/keadm/app/cmd/util/cloudinstaller.go

```diff
@@ -70,8 +74,21 @@ func (cu *KubeCloudInstTool) InstallTools() error {
 	return nil
 }
 
-//RunCloudCore starts cloudcore process
-func (cu *KubeCloudInstTool) RunCloudCore() error {
+func (cu *KubeCloudInstTool) RunCloudCoreContainer() error {
+	// start cloudcore image container
+	containerCommand := "for resource in $(ls *.yaml); do kubectl create -f $resource; done"
```

Comment:
Where this yamls come from? :）

Reply:
in source codes `build/cloud/01~08*.yaml`, include namspace, cluster role and etc.

Reply:
@gy95 So users need to download these manaully?

Reply:
yes, because if we run cloudcore in container, we should first install pre-requisites, and i think we can follow the guide of `Deploy the cloud part into a k8s cluster`, just create all needed k8s resources at once

Reply:
@gy95
I think `keadm init --run-mod=container` should do more, or this option is confusing to user.

Maybe it is time to start to release cloudcore-image and offer a standare and merged-into-one yaml .
/cc @fisherxu WDYT?

Reply:
yes, i think we can download the total.yaml from one website and apply it

Reply:
@zc2638 these yaml is used to init a basic cloudcore but not static, users need to do some configuration to these yaml for some high level feature such as `cloud stream`.

Reply:
@GsssC  I have different opinions.
In addition to the service configuration files, they are basically static files, and users do not need to modify them individually.
e.g. CRD resources、Deployment resources、certgen.sh

Reply:
so,you can use `command` generate the default configuration file. e.g. cloudcore.yaml

Reply:
CRD resources is certainly a static resources, but the others is volatile.
Deploying containerized cloudcore better like deploying `flannel`, `calico` and other k8s application.
```sh
curl cloudcore-deploy.yaml
kubectl apply -f cloudcore-deploy.yaml
```
`keadm` just download and apply `cloudcore-deploy.yaml'

Reply:
According to the feedback by the issues and wechat, cert always confues users, even if there is scripts about cert generation. So we change the way of cert generation from manual into auto. `CloudStream cert` also should generate automaticlly by code.

Reply:
It's a lot of trouble.

You should refer to `kubedm` method. We have specified the deployment method through the command, instead of personalized selection of CNI plugins after platform deployment

Reply:
@zc2638 I do not say personalized selection of CNI plugins after platform deployment, **but Deploying containerized cloudcore would better be like deploying flannel, calico and other k8s application**.

Reply:
@GsssC @zc2638 @fisherxu 
now, i use api interface to created prerequisite resources like ns, serviceaccount, and etc.(all yamls are in https://github.com/kubeedge/kubeedge/tree/master/build/cloud) when running cloudcore in container mode, 
but i have a question, now config file is mounted by configmap like this
https://github.com/kubeedge/kubeedge/blob/399b56a80fbfe381c6c4bb680bcd0feb35ca1a81/build/cloud/07-deployment.yaml#L48-L51
and configmap is from this 
https://github.com/kubeedge/kubeedge/blob/master/build/cloud/05-configmap.yaml
Could we mount directly using directory `/etc/kubeedge/config/cloudcore.yaml` to provide configuration.